integration/setup/setup.go

```diff
@@ -98,7 +99,27 @@ func Resources(g *framework.Guest, h *framework.Host) error {
 			return microerror.Mask(err)
 		}
 
-		err = h.InstallBranchOperator("kvm-operator", "kvmconfig", template.KVMOperatorChartValues)
```

Comment:
Should `template.KVMOperatorChartValues` be removed from the project?

Reply:
It is removed but is listed in the changes below as a `move` from template folder to the vendor. So it's moved but not expanded due to now being in the vendor.

Github being github :man_shrugging: